testdata/externalToken.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
Is this file correctly commited? What is it used for?

Reply:
Removed!